src/osd/OSD.cc

```diff
@@ -7376,8 +7350,9 @@ void OSDService::maybe_share_map(
                << session->projected_epoch << dendl;
       return;
     }
-
-    send_from = session->projected_epoch;
+    // send incremental maps in the range of:
+    // (projected_epoch, osdmap]
+    send_from = session->projected_epoch + 1;
```

Comment:
(not really part of this PR, but as I needed a moment to get over the indentation: )
I'd rather replace the block with 'unique_lock unlocked before the send_..

Reply:
Ack, can be addressed outside the scope of this PR.